sadu-updater/src/main/java/de/chojo/sadu/updater/SqlUpdater.java

```diff
@@ -185,7 +185,8 @@ public void init() throws IOException, SQLException {
             try {
                 performUpdate(patch);
             } catch (SQLException e) {
-                throw new UpdateException("Database update failed!", e);
+                String errorMessage = "Database update failed while applying patch " + patch.version().toString() + "!";
```

Comment:
Unused message
```suggestion
```

Reply:
Actually I thought that I have already removed this...
I may have messed up with GitHub Codespaces though. I am not used to VS Code (JetBrains Fanboi xD)